modules/apps/bookmarks/bookmarks-item-selector-web/src/main/java/com/liferay/bookmarks/item/selector/web/internal/display/context/BookmarksItemSelectorViewDisplayContext.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.bookmarks.item.selector.web.internal.display.context;
+
+import com.liferay.bookmarks.item.selector.criterion.BookmarksItemSelectorCriterion;
+import com.liferay.bookmarks.item.selector.web.internal.BookmarksItemSelectorView;
+import com.liferay.bookmarks.service.BookmarksEntryLocalService;
+import com.liferay.item.selector.ItemSelectorReturnTypeResolver;
+import com.liferay.item.selector.ItemSelectorReturnTypeResolverHandler;
+import com.liferay.petra.portlet.url.builder.PortletURLBuilder;
+import com.liferay.portal.kernel.portlet.LiferayPortletResponse;
+import com.liferay.portal.kernel.portlet.PortletURLUtil;
+import com.liferay.portal.kernel.repository.model.FileEntry;
+
+import java.util.Locale;
+
+import javax.portlet.PortletException;
+import javax.portlet.PortletURL;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @author Yang Cao
+ */
+public class BookmarksItemSelectorViewDisplayContext {
+
+	public BookmarksItemSelectorViewDisplayContext(
+		BookmarksItemSelectorCriterion bookmarksItemSelectorCriterion,
+		BookmarksItemSelectorView bookmarksItemSelectorView,
+		ItemSelectorReturnTypeResolverHandler
+			itemSelectorReturnTypeResolverHandler,
+		String itemSelectedEventName, boolean search, PortletURL portletURL,
+		BookmarksEntryLocalService bookmarksEntryLocalService) {
```

Comment:
Can you sort the constructor arguments alphabetically? Move BookmarksEntryLS just after the item selector view.